rwengine/src/render/TextRenderer.cpp

```diff
@@ -216,15 +200,17 @@ void TextRenderer::renderText(const TextRenderer::TextInfo& ti,
             switch (text[i + 1]) {
                 case 'b':  // Blue
                     text.erase(text.begin() + i, text.begin() + i + 3);
-                    colour = glm::vec3(glm::u8vec3(128, 167, 243)) * (1 / 255.f);
+                    colour =
+                        glm::vec3(glm::u8vec3(128, 167, 243)) * (1 / 255.f);
```

Comment:
nit: just wondering - won't `glm::vec3(128, 167, 243) * (1 / 255.f);` be enough?